docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -303,34 +306,34 @@ The following variables were renamed to have a coherent `Picker` / `Pickers` pre
 - `usePickersTranslations`
 
   ```diff
-  - import { usePickersTranslations } from '@mui/x-date-pickers/hooks';
-  - import { usePickersTranslations } from '@mui/x-date-pickers';
-  - import { usePickersTranslations } from '@mui/x-date-pickers-pro';
+  -import { usePickersTranslations } from '@mui/x-date-pickers/hooks';
+  -import { usePickersTranslations } from '@mui/x-date-pickers';
+  -import { usePickersTranslations } from '@mui/x-date-pickers-pro';
```

Comment:
Fix format

Reply:
Is there a way to lint such cases to avoid introducing incorrect diffs in the first place? 🤔

Reply:
It's possible yes. I did a first pass a long time ago https://github.com/mui/material-ui/blob/master/packages/markdownlint-rule-mui/git-diff.js. We could do another, I didn't think about this case. 

We can simply make sure that each diff resets the spacing, in a way that there is is no visual spacing offsets, e.g.

```diff
-    foo() {}
-      bar;
+foo() {}
+  bar;
```

Reply:
Moved to an issue https://github.com/mui/mui-public/issues/249.